FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#Unreleased
+- [fixed] Fix a crash related to registering for notifications when the app is inbetween foreground or background states. (#13174)
```

Comment:
nit: also add a space between the `#` and `Unreleased` so it matches the changelog updater script
```suggestion
# Unreleased
- [fixed] Fix a crash related to registering for notifications when the app is in between foreground or background states. (#13174)
```

Reply:
Done